sdk/assemblyscript/examples/vectors/asconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./node_modules/@hypermode/modus-sdk-as/plugin.asconfig.json",
   "options": {
-    "transform": ["@hypermode/modus-sdk-as/transform", "json-as/transform"]
+    "transform": ["@hypermode/modus-sdk-as/transform", "json-as/transform"],
+    "lib": ["./node_modules/json-as/lib"]
```

Comment:
Is this required?  It  builds fine without it, and it's not in the other examples.  If it's necessary, I'd prefer it was added in the common `plugin.asconfig.json` so users don't have to think about it.  But I don't think it's neccessary.

Reply:
Didn't realize that was left in the vectors example. No, its not required and I'll double check to make sure all references to lib are removed